src/python/TaskWorker/Actions/PreJob.py

```diff
@@ -535,6 +535,17 @@ def execute(self, *args):
         sleep_time = int(self.dag_retry)*60
         if old_time:
             sleep_time = int(max(1, sleep_time - old_time))
+        # Double check if it is HC job as they want to have finer submittion
+        if self.task_ad.get("CRAB_HC", '') and \
+           self.task_ad.get("CRAB_HC_JOB_TIMEOUT", '0') and \
```

Comment:
I am not sure I would like to bind this functionality to HC. Let's have a generic name for this classad for slow release. Also, when the name is picked up we should document this feature, @emaszs can you do it? Maybe here https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3TipsAndTricks is a good place @belforte ?